### DISCUSSION TITLE
call_peak GRanges object

### DISCUSSION THREAD

**Discussion by AnjaliC4** [stuart-lab/signac#829] [Q&A] [answered]
Hi @timoast ,

The CallPeak function in Signac does not remove the unknown chromosomes such as "chr9_KQ090019v1_alt" or the blacklist regions. I was wondering if there was a straightforward way to subset the output Granges object to keep only the standard chromosomes and without the blacklisted regions.

Thank you,
Anjali

**Comment by timoast** [comment, marked as answer]
You can use the `keepSeqlevels()` or `keepStandardChromosomes()` functions in the `GenomeInfoDb` package to remove chromosomes from the genomic ranges returned by `CallPeaks()`
